docs/entity/components.md

```diff
@@ -94,6 +94,10 @@ A clear button is visible to the right of the dropdown when this field is marked
 | valueField                                        | string  | If you use endpointUrl and your data are not simple text data, you can specify here which property of retrieved object should be used as value for each item.```item.content?.[valueField]``` | -             |
 | [dependencies](../advanced/dependent_dropdown.md) | array   | It is used to update the options via an API call when the value of any field in the dependencies list is updated.                                                                             | -             |
 
+> When using [Boolean](https://docs.splunk.com/Documentation/SplunkCloud/9.2.2403/SearchReference/ListOfDataTypes) values consider that inside splunk values like 'TRUE', 'T', 'Y', 'YES', true will be converted into '1' and values like 'FALSE', 'F', 'N', 'NO', 'NONE', false will be converted into '0'.
```

Comment:
We can keep [Splunk enterprise](https://docs.splunk.com/Documentation/Splunk/latest/SearchReference/ListOfDataTypes) and [Splunk Cloud](https://docs.splunk.com/Documentation/SplunkCloud/latest/SearchReference/ListOfDataTypes) reference links pointing to "latest" version and not a specific version.

Reply:
oh, nice spot, fixed